src/main/java/com/google/jenkins/plugins/k8sengine/KubernetesEngineBuilder.java

```diff
@@ -284,6 +293,48 @@ private boolean verify(
         kubectl, manifestObjects, consoleLogger, verifyTimeoutInMinutes);
   }
 
+  /**
+   * @param credentialsId The service account credential's id.
+   * @return Access token from OAuth to allow kubectl to interact with the cluster.
+   * @throws IOException If an error occurred fetching the access token.
+   */
+  public static String getAccessToken(String credentialsId) throws IOException {
+    Preconditions.checkArgument(!Strings.isNullOrEmpty(credentialsId));
+    HttpTransport HTTP_TRANSPORT = new NetHttpTransport();
+    JsonFactory JSON_FACTORY = new JacksonFactory();
+
+    // get the credential from jenkins
```

Comment:
I don't think this comment is needed here.

Reply:
done.